argoverse/utils/ply_loader.py

```diff
@@ -26,3 +26,37 @@ def load_ply(ply_fpath: _PathLike) -> np.ndarray:
     z = np.array(data.points.z)[:, np.newaxis]
 
     return np.concatenate((x, y, z), axis=1)
+
+
+def load_ply_by_attrib(ply_fpath: _PathLike, attrib_spec: str) -> Optional[np.ndarray]:
```

Comment:
Should we default `attrib_spec` to `attrib_spec = "xyzil"`?

Reply:
i like that

Reply:
updated